platform/posix/utest/reconnect_utest.c

```diff
@@ -67,6 +67,7 @@ static void verifyReconnectParamsAfterReset( void )
  */
 void test_Transport_ReconnectParamsReset_Sets_Jitter_Correctly( void )
 {
+    clock_gettime_ExpectAnyArgsAndReturn( 0 );
```

Comment:
I need to expect these now in the reconnect unit tests as they are being mocked.